packages/node/src/chunk-engine.ts

```diff
@@ -1,40 +0,0 @@
-import { mergeCss } from '@pandacss/core'
-import type { Generator } from '@pandacss/generator'
-import type { Artifact, PandaHookable } from '@pandacss/types'
-import type { Runtime } from '@pandacss/types/src/runtime'
-
-export const getChunkEngine = ({
-  paths,
-  config,
-  runtime: { path, fs },
-}: Generator & { runtime: Runtime; hooks: PandaHookable }) => ({
-  dir: path.join(...paths.chunk),
-  readFile(file: string) {
-    const fileName = path.join(...paths.chunk, this.format(file))
-    return fs.existsSync(fileName) ? fs.readFileSync(fileName) : ''
-  },
-  getFiles() {
-    const files = fs.existsSync(this.dir) ? fs.readDirSync(this.dir) : []
-    return files.map((file) => fs.readFileSync(path.join(this.dir, file)))
-  },
-  format(file: string) {
-    return path.relative(config.cwd, file).replaceAll(path.sep, '__').replace(path.extname(file), '.css')
-  },
-  getArtifact(file: string, css: string): Artifact {
-    const fileName = this.format(file)
-    const newCss = mergeCss(this.readFile(file), css)
```

Comment:
This line was added previously to avoid janky updates or unstyled flashes when using the CLI. We need to recreate this with the in-memory version.

It should be essentially similar to the Builder used in postcss.

Reply:
I thought so too but I couldn't reproduce this on the panda website locally, is there a way to reproduce this ?

Reply:
You can reproduce this by running any of the sandbox examples with the CLI watch flag. No postcss